src/main/java/frc/robot/Operator.java

```diff
@@ -7,15 +7,25 @@
     private Controller OP;
     private Wheels wheels;
     private ControlPanel controlPanel;
+    private Climber climb;
 
     private boolean startedPositionControl;
     private boolean finishedPositionControl;
     
+    private long lastButtonPress;//Last time rotation control button was pressed
+    private final int BUTTON_DELAY = 500;//Delay time for rotation control button in ms
+    
+    private long lastPositionPress;//Last time position control button was pressed
+    private final int POSITION_TIMEOUT = 1000;//Position control timeout in ms
+    
     public Operator(int port) {
         OP = new Controller(port);
         wheels = new Wheels();
         controlPanel = new ControlPanel();
+        climb = new Climber();
+        
         resetControlPanel();
+        lastButtonPress = System.currentTimeMillis();
     }
 
     public void opControls() {
```

Comment:
This should be in driver controls, not operator

Reply:
Why? or what should be?

Reply:
The climber code should be in Driver.java

Reply:
Control panel stays in Operator.java, climber code goes to Driver.java -- then you're good

Reply:
@ZekromZZ said to put it on operator controls.

Reply:
Kat said to put it in driver controls and I agree based off of my experience behind the glass. Climbing is too risky of an operation to allow for possible communication barriers between driver and operator.